assets/stylesheets/components/_sticky.scss

```diff
@@ -0,0 +1,11 @@
+.sticky-example-wrapper {
```

Comment:
@galiotoweb This looks like it shouldn't belong here. Perhaps this should go into crdschurch/crds-styleguide#287?

Reply:
Yep. I moved it.